internal/info/info_bench_test.go

```diff
@@ -0,0 +1,119 @@
+package info
+
+import (
+	"testing"
+)
+
+func BenchmarkString(b *testing.B) {
+	Init("benchmark")
```

Comment:
Should we check nil or not nil of `infoProvider ` just in case?

Reply:
I think it is not needed as it should be covered in unit test.

Reply:
I got it. thank you 🙏